releaseNote.md

```diff
@@ -0,0 +1,48 @@
+
+
+# Release Notes - [Version 2.12]
+
+## Overview
+
+In this release, we've implemented several updates and enhancements to improve functionality and compatibility in the project. Please review the following changes and take note of important considerations:
+
+### 1. Removal of Deprecated Packages
+
+The `mrest:restivus` package has been removed from the project due to its lack of support and incompatibility with the latest Meteor version. This removal ensures a streamlined and up-to-date environment for your project.
+
+### 2. Python 2 Compatibility
+
+For optimal performance, we recommend using Python 2 with this project. Please ensure your Python environment is set to version 2 to ensure compatibility and a seamless operation.
+
+### 3. Elimination of Unsupported Package
+
+We have removed the `cultofcoders:persistent-session` package from the project as it is no longer supported and compatible with the latest Meteor version. This step guarantees that the project remains compatible and well-maintained.
+
+### 4. Handling of Role 'admin'
+
+An important update relates to the handling of the 'admin' role. During the account creation process, you might encounter an error message: "Exception while invoking method 'account.check': Error: Role 'admin' does not exist." This error is attributed to a change in the `alanning:roles` package, which no longer supports the 'admin' role by default.
+
+## Action Required
+
+To ensure a smooth transition and optimal performance after this update, please take the following actions:
+
+1. **Remove Deprecated Packages**: Remove the `mrest:restivus` and `cultofcoders:persistent-session` packages from your project to prevent compatibility issues.
```

Comment:
since this `cultofcoders:persistent-session` is used in some places, can we replace it's usage with Meteor/session package.  example which is similar to others is this https://github.com/SparkEdUAB/SparkEd/blob/830094e1d0394fe43f123cc53e067811d2bad474/imports/ui/components/Dashboard/Courses.jsx#L402

Ref: https://github.com/search?q=repo%3ASparkEdUAB%2FSparkEd%20Session.setPersistent&type=code

Reply:
Noted. Let me do that.

Reply:
@OlivierJM I have made the requested changes